react/components/molecules/checkbox-group/checkbox-group.stories.js

```diff
@@ -0,0 +1,101 @@
+import React, { PureComponent } from "react";
+import { Text, View } from "react-native";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, boolean } from "@storybook/addon-knobs";
+import PropTypes from "prop-types";

```

Comment:
I would reorder this (storybook stuff after `PropTypes`):
```suggestion
import PropTypes from "prop-types";
import { storiesOf } from "@storybook/react-native";
import { withKnobs, boolean } from "@storybook/addon-knobs";
```